Mesh_3/include/CGAL/Mesh_3/C3T3_helpers.h

```diff
@@ -1514,7 +1514,8 @@ class C3T3_helpers
           c3t3.remove_from_complex(Facet(c,i));
 
         c->set_facet_surface_center(i, facet_surface_center_[old_i]);
-        c->set_facet_surface_center_index(i, surface_center_index_table_[old_i]);
```

Comment:
why do you remove this?

Reply:
Because the code `c3t3.add_to_complex(Facet(c, i), index);` or `c3t3.remove_from_complex(Facet(c,i));`, a few line above, has already restored the index, using the API of the C3t3.

Reply:
ok!